src/plugins/ovp/components/livestreams-screen/LivestreamsCard.jsx

```diff
@@ -108,23 +110,31 @@ const LivestreamsCard = ({ nodes, metas, reloadChannelState }) => nodes.map((nod
         .set('node_ref', nodeRef);
       const pbjx = await app.getPbjx();
       await pbjx.send(command);
-      await delay(5000);
-      reloadChannelState();
-      await progressIndicator.close();
-      await delay(2000);
-      dispatch(sendAlert({
-       type: 'success',
-       isDismissible: true,
-       delay: 3000,
-       message: 'Success! The MediaLive Channel was started.',
-      }));
-    }catch(error){
-      await progressIndicator.close();
+
+      const eventName = 'triniti:ovp.medialive:event:channel-started.raven';
+      const dispatcher = app.getDispatcher();
+      const cleanup = () => {
+        progressIndicator.close();
+        delay(2000);
```

Comment:
@meghedi this delay is not necessary anymore, right? especially since it is not even being awaited

Reply:
Done.